### PR TITLE
Docs: minor fixes to AWS EKS and AWS Metadata filtering GSGs

### PR DESCRIPTION
**Pull request by danwent**
Minor changes based on recent testing of AWS metadata filtering using an EKS cluster.

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/7856)

### CODE REVIEW

**Comment by danwent** [comment]
test-me-please

**Comment by ianvernon** [comment]
test-docs-please

**Comment by ianvernon** [comment]
test-docs-please

File: Documentation/gettingstarted/aws.rst [hunk 1]
```diff
@@ -116,16 +122,50 @@ To list all of the available AWS instances, the following command can be used:
 
    $ kubectl  -n kube-system exec -ti testing-aws-pod -- aws ec2 describe-instances
 
-It is important for this demo that ``kube-dns`` is working correctly. To know the
-status of ``kube-dns`` you can run the following command:
+Once the secret has been created and validated, the cilium-operator pod must be
+restarted in order to pick up the credentials in the secret.
+To do this, identify and delete the existing cilium-operator pod, which will
```
Comment (nebril) [inline Documentation/gettingstarted/aws.rst]: s/which will/which will be

File: Documentation/gettingstarted/aws.rst [hunk 2]
```diff
@@ -116,16 +122,50 @@ To list all of the available AWS instances, the following command can be used:
...
+group ('sg-0f2146100a88d03c3') and pods with label class=xwing should
+only be able to make connections outside the cluster to the destination
+VMs in that security group.
+
+To enable this, the VMs acting as Kubernetes worker nodes must be able to
+send traffic to the destination VMs that are being accessed by pods.  One approach
+for achieving this is to put all Kubernetes worker VMs in a single 'k8s-worker'
+security group, and then ensure that any security group that is referenced in a
+Cilium toGroups policy has an allow all ingress rule (all ports) for connections from the
+'k8s-worker' security group.    Cilium filtering will then ensure that the only pods allowed
```
Comment (nebril) [inline Documentation/gettingstarted/aws.rst]: Will multiple spaces before "Cilium" render? If yes, I think they should be deleted.

**Review by nebril — CHANGES_REQUESTED** [review]
LGTM overall, two comments inline

**Comment by danwent** [comment]
Thanks @nebril , made the changes you suggested, and added sign-off.   Thanks!

**Comment by aanm** [comment]
test-docs-please

**Comment by danwent** [comment]
@nebril  , can you please take a look and confirm that your comments have been addressed?   thanks!

**Review by tgraf — APPROVED** [review]

**Pull request merged**